tests/php/Functional/WC/Gateway/Mollie_WC_Gateway_Abstract_Test.php

```diff
@@ -14,7 +14,7 @@ class Mollie_WC_Gateway_Abstract_Test extends TestCase
     protected function setUp()
     {
         parent::setUp();
-
+        require('inc/woocommerce.php');
```

Comment:
Always use absolute path plus, this can be moved into `setUpBeforeClass`, you save some calls among tests.
Also this is a statement not a function in php, you don't need to use parenthesis.

See https://phpunit.readthedocs.io/en/8.5/fixtures.html#sharing-fixture